basics/Main.java

```diff
@@ -1,5 +1,61 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
 public class Main {
   public static void main(String[] args) {
-    System.out.println("Hello");
+    // pluralize tests
+    int dogCount = 1;
+    System.out.println("I own " + dogCount + " " + pluralize("dog", dogCount) + ".");
+    int catCount = 2;
+    System.out.println("I own " + catCount + " " + pluralize("cat", catCount) + ".");
+    int turtleCount = 0;
+    System.out.println("I own " + turtleCount + " " + pluralize("turtle", turtleCount) + ".");
+
+    // flipping coins tests
+    flipNHeads(4);
+    flipNHeads(1);
+    flipNHeads(6);
+
+    // clock run
+    clock();
   }
-}
\ No newline at end of file
+
+  // pluralize function
+  public static String pluralize(String word, int count) {
+    if (count == 0 || count > 1) {
```

Comment:
Your edge case is anything that's not a singular word.  So knowing that, you could just do
if (count != 0) { return word + "s"} else { return word }